### ISSUE TITLE
Fix crash in the port library

### ISSUE THREAD

**Issue by gita-omr** [eclipse-openj9/openj9#13897]
Looks like some unknown failure during PR test:

https://openj9-jenkins.osuosl.org/job/Build_JDK17_ppc64le_linux_Personal/68/console

**Comment by pshipton** [comment]
The crash is not in the port library, it's in the JIT. Also the link will quickly expire, making this Issue useless.

If it's failing in PR testing, and we haven't seen this crash before, how do we know the PR being tested isn't causing the crash?

**Comment by gita-omr** [comment]
The change in the PR can only affect methods that use Vector API. I think the method being compiled does not use Vector API.

**Comment by gita-omr** [comment]
I added some more context into the description in case the link expires before we can get to investigate this failure, which most likely is very intermittent.

**Comment by pshipton** [comment]
Another crash on plinux https://github.com/eclipse-openj9/openj9/issues/13906

**Comment by 0xdaryl** [comment]
@zl-wang : there have been some Power Linux crashes appearing in the builds over the past couple of days.  Could you have someone triage please?

**Comment by zl-wang** [comment]
@IBMJimmyk please take a look

**Comment by pshipton** [comment]
The crash is with the boot JVM, as follows. There have been issues fixed since this build, if necessary we can update the boot JVM being used.
```
1CIJAVAVERSION JRE 17 Linux ppc64le-64 (build 17+35)
1CIVMVERSION   20210916_35
1CIJ9VMTAG     openj9-0.28.0-m2
```

**Comment by gita-omr** [comment]
Yes, DDR is built with boot JDK. To add debug options, use DDR_TOOLS_OPTIONS in DDR.gmk.

**Comment by pshipton** [comment]
@AdamBrousseau can we update the JDK17 boot JVM to the 0.29.1 M1 pls.
